konfera/utils.py

```diff
@@ -1,8 +1,18 @@
+import re
+import logging
 from decimal import Decimal, ROUND_UP
+from smtplib import SMTPException
 
+from django.core.mail import EmailMultiAlternatives
+
+from konfera.models.email_template import EmailTemplate
 from konfera.models.sponsor import Sponsor
 from konfera.models.order import Order
 from konfera.settings import GOOGLE_ANALYTICS, NAVIGATION_ENABLED, NAVIGATION_URL, NAVIGATION_LOGO, NAVIGATION_BRAND
+from konfera.settings import EMAIL_NOTIFY_BCC
```

Comment:
Why do you need to import twice from the same file, for multiline imports use brackets...